### PR TITLE
Javascript javascript2 week3

### PR DESCRIPTION
**Pull request by saloumeh-67**
I have problem with restarting game ! i have no idea why its not working.

### CODE REVIEW

File: javascript/javascript2/week3/js2week3.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  transform-origin: 0 0;
+  transition: transform .6s;
+  user-select: none;
+  font-size: 30px;
+}
+
+    </style>
+</head>
+<body id = "dclk">
+<button id="click">click Here</button>
```
Comment (Lupescua) [inline javascript/javascript2/week3/js2week3.html:26]: Looks good. 
1. One thing you could maybe make better is a better naming convention. The id="click" would be confusing in a bigger project. I'd suggest and id="delay5Sec"
2. The id before "dclick" is a better naming convention... though.. the body should not have an id .. but yeah. good work around

File: javascript/javascript2/week3/Game.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+const input = document.getElementById('input');
+const button = document.getElementById('btn');
+const printL = document.getElementById('printL');
+const printS = document.getElementById('printS');
+const pressKey = document.querySelector('body');
+let counterS = 0;
+let counterL = 0;
+const winnerMsg = document.getElementById('winner');
+const restart = document.getElementById('restart');
```
Comment (Lupescua) [inline javascript/javascript2/week3/Game.js:9]: Great start with the constants at the top

File: javascript/javascript2/week3/Game.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+const print = function (e) {
+    if (e.key == 'l') {
+        counterL++
+        printL.innerText = counterL
+    } else if (e.key == 's') {
+        counterS++
+        printS.innerText = counterS
+    };
+};
+const game = function () {
```
Comment (Lupescua) [inline javascript/javascript2/week3/Game.js:28]: I would use this this other way

function game() {}

File: javascript/javascript2/week3/Game.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    let timeCounter = durationTime();
+
+    const timerPrint = function () {
+        console.log('start');
+        if (timeCounter >= 0) {
+            timer.innerHTML = timeCounter
+            timeCounter--;
+        }
+    }
+    button.addEventListener('click', game);
```
Comment (Lupescua) [inline javascript/javascript2/week3/Game.js:50]: I am not sure I know why the button event listener is inside. i think you meant to put it outside the game function. Isn't this the way you call your function?

File: javascript/javascript2/week3/Game.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    if (e.key == 'l') {
+        counterL++
+        printL.innerText = counterL
+    } else if (e.key == 's') {
+        counterS++
+        printS.innerText = counterS
+    };
+};
+const game = function () {
+    pressKey.addEventListener('keydown', print);
```
Comment (Lupescua) [inline javascript/javascript2/week3/Game.js:29]: great choice with keydown. Prevents users from just holding the key

File: javascript/javascript2/week3/Game.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+        console.log('start');
+        if (timeCounter >= 0) {
+            timer.innerHTML = timeCounter
+            timeCounter--;
+        }
+    }
+    button.addEventListener('click', game);
+
+    /////restart function
+    restart.addEventListener('click', restartGame);
```
Comment (Lupescua) [inline javascript/javascript2/week3/Game.js:53]: I think you'll want the restart function to be a separate function, outside the game()

File: javascript/javascript2/week3/Game.js [hunk 7]
```diff
@@ -0,0 +1,62 @@
+const input = document.getElementById('input');
```
Comment (Lupescua) [inline javascript/javascript2/week3/Game.js:1]: The Game does not seem to work ..

**Review by Lupescua — COMMENTED** [review]
Great job :D

You know a lot ! :D

I mostly gave aesthetic feedback and some more on the Game.js.. I'd have one more feedback. I want you to get a superpower in coding: Commenting. 

You don't have to write clean or even good code. But if you can write great comments, everyone would appreciate you and your work 1000 times more. Imagine that when your future self comes back to some old code, you can understand why you wrote what you did in just a few seconds. Believe me.. it's better than magic.

Your current tasks don't allow you much flexibility in that sense, but you could start with putting the exercise text before each solution. Similar to what you did on your functions.js

I'll be honest.. I would have totally appreciated more comments on the Game.js :D To understand you and why you chose that way of doing things. FX: line 29.. or even better line 50.

Anyways! Good job

File: javascript/javascript2/week3/Game.js (hunk 4 above)
Comment (saloumeh-67) [reply javascript/javascript2/week3/Game.js:50]: yes

**Pull request merged**